actions/testabilities/assert_spvwallet_application.go

```diff
@@ -90,6 +98,18 @@ func (a *responseAssertions) assertJSONBody(expectedFormat string, args ...any)
 	assertJSONEq(a.t, expectedJson, a.response.String())
 }
 
+func (a *responseAssertions) assertJSONTemplate(expectedTemplateFormat string, params map[string]any) {
```

Comment:
I moved it to separate `jsonrequire` package.

But about `testingT` is faced some problems so it would be nice to adjust it in some other step (PR) 🙏